tests/timers.spec.ts

```diff
@@ -79,8 +74,7 @@ describe("$after", () => {
             eventTimestamp: 8,
         })
 
-        validate(timers)
-
+        assert.strictEqual(timers.length, 0, "incorrect timer count")
         assert.strictEqual(result2, true, "timer returned false")
     })
```

Comment:
Can we make a new test for this fix? The library is already a bit of a mess, so 100% code coverage is something that I'm really aiming for.